client/jetpack-connect/plans.jsx

```diff
@@ -47,7 +47,7 @@ import {
 	isCalypsoStartedConnection,
 } from 'state/jetpack-connect/selectors';
 import { mc } from 'lib/analytics';
-import { isJetpackSite } from 'state/sites/selectors';
+import { isCurrentPlanPaid, isJetpackSite } from 'state/sites/selectors';
```

Comment:
Nearly duplicated selectors, this one happens to return `null` with missing data. I don't know if that's 🤷‍♂️  or 🤦‍♂️